docs/reference/experiment-config.txt

```diff
@@ -605,6 +605,13 @@ created with the Determined CLI. It may contain the following fields:
     ``tensor_fusion_threshold`` and ``tensor_fusion_cycle_time`` automatically.
     Defaults to ``false``.
 
+  - ``horovod_timeline``: The path where the Horovod should be timeline saved. The
+    Horovod timeline is a trace file that can be analyzed to investigate distributed
+    training performance. When this field is ``None`` or an empty string, the timeline is not created.
```

Comment:
Should be `null`, no? i.e., YAML doesn't have `None`.

Reply:
Fixed